server/config/config.go

```diff
@@ -776,6 +776,9 @@ type ScheduleConfig struct {
 
 	// EnableWitness is the option to enable using witness
 	EnableWitness bool `toml:"enable-witness" json:"enable-witness,string"`
+
+	// HotThresholdRatio is used to calculate hot thresholds
+	HotThresholdRatio float64 `toml:"hot-threshold-ratio" json:"hot-threshold-ratio,omitempty"`
```

Comment:
Do we need to validate the value of this config?

Reply:
add non-negative check